# Ros Topics
# image0_topic: "/left_r200/camera/color/image_raw"
# image1_topic: "/right_r200/camera/color/image_raw"

image0_topic: "/camera/left/image_raw"
image1_topic: "/camera/right/image_raw"

visualize: true
useGrav: true
# Intrinsic Camera Paramters
fx: 4.6115862106007575e+02
fy: 4.5975286598073296e+02
cx: 3.6265929181685937e+02
cy: 2.4852105668448124e+02
k1: -2.9545645106987750e-01
k2: 8.6623215640186171e-02
p1: 2.0132892276082517e-06
p2: 1.3924531371276508e-05
k3: 0.0
width: 752
height: 480

# Extrinsic Camera Paramters 
baseline: 0.03
estimators: [1,2,3]  #1 , 2 ,3 ONLY 1 ESTIMATOR, 4 IS Up down, 5 is all 3
# initX: 3.75
# initY: 0.0
# initZ: 1.25
# initRoll: 0.0
# initPitch: 0.0
# initYaw: 1.570796 
initX: 0.0
initY: 0.0 #0
initZ: 0.0 #1.0104

initRoll: 0.0
initPitch: 0 # 0.4 #0.62 #0.0
initYaw: 0 # -0.15
gravMag: 9.81
#gravMag: 1.0